src/main/java/com/minecolonies/coremod/colony/requestsystem/management/handlers/ResolverHandler.java

```diff
@@ -135,16 +139,16 @@ public static void removeResolver(final IStandardRequestManager manager, final I
      * @param resolver The resolver to remove
      * @throws IllegalArgumentException is thrown when the given resolver is not registered or the token of the given resolver is not registered to the same resolver.
      */
-    public static void removeResolver(final IStandardRequestManager manager, final IRequestResolver resolver) throws IllegalArgumentException
+    public static void removeResolver(final IStandardRequestManager manager, final IRequestResolver<?> resolver)
     {
-        final IRequestResolver registeredResolver = getResolver(manager, resolver.getRequesterId());
+        final IRequestResolver<?> registeredResolver = getResolver(manager, resolver.getRequesterId());
 
         if (!registeredResolver.equals(resolver))
         {
             throw new IllegalArgumentException("The given resolver and the resolver that is registered with its token are not the same.");
         }
 
-        if (manager.getResolverRequestMap().containsKey(registeredResolver.getRequesterId()) && manager.getResolverRequestMap().get(registeredResolver.getRequesterId()).size() > 0)
+        if (manager.getRequestResolverRequestAssignmentDataStore().getAssignments().containsKey(registeredResolver.getRequesterId()) && manager.getRequestResolverRequestAssignmentDataStore().getAssignments().get(registeredResolver.getRequesterId()).size() > 0)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 260 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)